install_plugin_only.sh

```
@@ -0,0 +1,120 @@
+if [ ! -f "api_server.py" ]; then
+    echo "❌ Error: Please run this script from the AIIngestingTool directory"
+    echo "   Expected: /Users/developer/Development/GitHub/AIIngestingTool"
+    exit 1
+fi
+
+echo "📋 Current directory: $(pwd)"
+
+# Step 1: Download required CEP libraries if missing
+echo ""
+echo "📚 Checking CEP libraries..."
+cd premiere_pro_panel/js/lib
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)